docs/config/apache.md

```diff
@@ -16,28 +16,28 @@ description: Add an Apache webserver service to your Lando config for local deve
 ## Patch versions
 
 ::: warning Not officially supported!
-While we allow users to specify patch versions for this service they are not *officially* supported so if you use one YMMV.
+While we allow users to specify patch versions for this service, they are not *officially* supported, so if you use one, YMMV.
 :::
 
-To use a patch version you can do something like this:
+To use a patch version, you can do something as shown below:
 
 ```yaml
 services:
-  my-service:
+  myservice:
```

Comment:
@uberhacker im not sure i understand the change here? is `my-service` not a valid service name?

Reply:
It's been my experience that using non-alphanumeric characters creates pain.  Docker prefers underscores, correct?  I just thought it would be better for consistency throughout the entire naming process.